core/src/executor/evaluate/function.rs

```diff
@@ -471,3 +471,14 @@ pub fn to_time(name: String, expr: Evaluated<'_>, format: Evaluated<'_>) -> Resu
         _ => Err(EvaluateError::FunctionRequiresStringValue(name).into()),
     }
 }
+
+pub fn position(name: String, expr: Evaluated<'_>, format: Evaluated<'_>) -> Result<Value> {
+    match expr.try_into()? {
+        Value::Str(_) => {
+            let format = eval_to_str!(name, format);
+            let num = format.parse::<i64>().unwrap();
+            Ok(Value::I64(num))
```

Comment:
See <https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.position>
The index cannot be negative, how about using size (8 bytes)?

Reply:
thanks for your careful review